### PR TITLE
Feat: UX improvements to Control Plane Groups

### PR DESCRIPTION
**Pull request by hl782**
change log for DOCU-3677
Feat: UX improvements to Control Plane Groups

### Testing instructions

Preview link: 

### Checklist 

- [x] Review label added 
- [ ] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

For example, if this change is for an upcoming 3.6 release, enclose your content in `{% if_version gte:3.6.x %} <content> {% endif_version %}` tags (or `if_plugin_version` tags for plugins). 

Use any of the following keys:
* `gte:<version>` - greater than or equal to a specific version
* `lte:<version>` - less than or equal to a specific version
* `eq:<version>` - exactly equal to a specific version

You can do the same for older versions.

### CODE REVIEW

**Review by acgoldsmith — APPROVED** [review]
LGMT

**Comment by acgoldsmith** [comment]
@Guaris I approved this one before I saw it was internal release request, so have put back in draft mode until the GA request.

File: app/konnect/updates.md [hunk 1]
```diff
@@ -11,7 +11,10 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 ## March 2024
 
-**Enhanced error notification in {{site.konnect_short_name}}**
+**Refined user experience for Control Plane Groups**
+: We've made improvements to refine the end-to-end workflow of Control Plane Groups (CPGs) for our customers. Now, CPGs are more intuitive, and easier to understand.
```
Comment (rosskukulinski) [inline app/konnect/updates.md:15]: I would suggest not shortening in the release note. Just spell out Control Plane Groups

Comment (Guaris) [inline app/konnect/updates.md:15]:
```suggestion
: We've made improvements to refine the end-to-end workflow of Control Plane Groups for our customers. Now, Control Plane Groups are more intuitive, and easier to understand.
```

Comment (Guaris) [reply app/konnect/updates.md]: @rosskukulinski done

**Pull request merged**